spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
Given the above workflow, why does the CO care that the provisioned volume is actually `MULTI_NODE_READER_ONLY` vs. what was requested? Is it trying to make decisions about other workloads on other nodes that may be trying to use the same volume (and if so, shouldn't it stop them from doing so because step 1 requested `SINGLE_NODE_READER_ONLY` to begin with..)?